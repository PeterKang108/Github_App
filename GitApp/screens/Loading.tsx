import React from "react"
import { Text } from '../components/Themed'
import { StyleSheet ,Text as DefaultText} from 'react-native';

export default (props:DefaultText['props'])=>{
    return (
        <Text {...props}></Text>
    )
}