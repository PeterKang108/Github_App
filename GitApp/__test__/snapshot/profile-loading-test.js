
import renderer from 'react-test-renderer'
import * as React from "react"
import Loading from "../../components/parts/Loading"

it('Profile Loading Views correctly',async ()=>{
    let tree = renderer.create(<Loading>Loading</Loading>).toJSON()
    expect(tree).toMatchSnapshot()
})