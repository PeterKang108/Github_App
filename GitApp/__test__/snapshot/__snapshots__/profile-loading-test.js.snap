// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Views correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Object {
        "height": 1,
        "marginVertical": 30,
        "width": "80%",
      },
    ]
  }
>
  Loading
</Text>
`;

exports[`Profile Loading Views correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Object {
        "height": 1,
        "marginVertical": 30,
        "width": "80%",
      },
    ]
  }
>
  Loading
</Text>
`;
